components/nimbus/src/nimbus.udl

```diff
@@ -110,9 +104,12 @@ interface NimbusClient {
     [Throws=NimbusError]
     string? get_experiment_branch(string id);
 
+    [Throws=NimbusError]
+    string? get_feature_config_variables(string feature_id);
```

Comment:
Is this waiting on a new uniffi dependency in order to switch to a JSONValue?  If so, will that happen in this PR or a different one?

Reply:
I don't think this is going to get a JSON Value in time, so we're dropping down to using a `String`. :(